File: 1. Basic Programming/1. Basics of Input Output/Python/seating_arrangement.py
t = int(input())
while(t):
    t -= 1
    s = int(input())
    win = s%6
    wins = ""
    if(win == 1 or win == 0):
        wins = "WS"
    elif(win == 2 or win == 5):
        wins = "MS"
    else:
        wins = "AS"
    fron = s%12
    if(fron == 0):
        fron = 12
    frons = s + 13 - (2*fron)
    print(frons,wins)

File: 1. Basic Programming/1. Basics of Input Output/C++/seating_arrangement.cpp
#include<bits/stdc++.h>
using namespace std;

int main()
{
    int t = 0;
    cin>>t;
    while(t--)
    {
        int s = 0;
        cin>>s;
        int win  = s%6;
        string wins = "";
        if(win == 1 || win == 0)
            wins = "WS";
        else if(win == 2 || win == 5)
            wins = "MS";
        else
            wins = "AS";
        int fron = s%12;
        if(fron == 0) 
            fron = 12;
        cout<<s+13-(2*fron)<<" "<<wins<<endl; 
    }
    return 0;
}
